### PR TITLE
Add group to domains that can be toggled

### LINKED ISSUE #1819: Header toggle doesn't work on groups
In Home Assistant (in 0.80.3 - but also earlier versions too) when I switch a whole card on, the light domains switch, but the group domains don't. That means that when I switch the whole card on, groups are leaved untouched and therefore some things aren't turned on.

Other bug is that group domains don't light up (turn yellow) when they're turned on, only lights. That leads to a odd appearance on the UI.

I made a GIF to show what I mean:

groupsbug

### PR DESCRIPTION
**Pull request by bramkragten**
Closes https://github.com/home-assistant/home-assistant-polymer/issues/1819

### CODE REVIEW

File: src/common/const.ts [hunk 1]
```diff
@@ -77,6 +77,7 @@ export const DOMAINS_TOGGLE = new Set([
   "input_boolean",
   "light",
   "switch",
+  "group",
```
Comment (balloob) [inline src/common/const.ts]: We shouldn't do that, we should consider using `canToggleDomain` and `canToggleState`

Comment (bramkragten) [reply src/common/const.ts]: I think I’m missing something, what would they do?

Comment (balloob) [reply src/common/const.ts]: Oh wait, is this just for the services being called?

Comment (bramkragten) [reply src/common/const.ts]: It seems to be used in 3 places: to check if a `group-entity-row` should show display a toggle (was a bug in that), to see what entities should be toggled in `hui-entities-toggle` and what should be done in `picture-glance`, toggle or dialog. The last one might be a problem...

**Pull request merged**